python/ray/data/aggregate.py

```diff
@@ -534,7 +703,17 @@ def _finalize(self, accumulator: List[Any]) -> Optional[U]:
 
 @PublicAPI
 class Unique(AggregateFnV2):
-    """Defines unique aggregation."""
+    """Defines unique aggregation.
+
+    Usage:
+        `ds.groupby("group_key").aggregate(Unique(on="value_col"))`
```

Comment:
For each of these; can you just add a full example? 

The standard way to do this is with an `Example:` heading, and using a code block like `.. testcode`

Reply:
updated